src/login/lclif.c

```diff
@@ -293,6 +293,7 @@ static bool lclif_send_server_list(struct login_session_data *sd)
 	packet->packet_id = HEADER_AC_ACCEPT_LOGIN;
 #else
 	packet->packet_id = HEADER_AC_ACCEPT_LOGIN2;
+	login->generate_token(sd, packet->auth_token);
```

Comment:
here need add packet version check. because in old packet version field ``auth_token`` not exists and will be compilation error

Reply:
This line is inside the `#else` of `#if PACKETVER < 20170315` so it should be compiling correctly. Also Github's checks api hasn't found any problem when compiling.